src/plone/restapi/tests/test_translations.py

```diff
@@ -81,7 +81,7 @@ def setUp(self):
         )
         transaction.commit()
 
-    def test_translation_linking_succeeds(self):
+    def test_translation_linking_by_url_succeeds(self):
```

Comment:
@erral nitpicking: I'd remove the "_succeeds" here since it does not add any kind of information IMHO. Not your fault. You just continued to use what was there already.

Reply:
done :)